src/NuGet.Clients/NuGet.PackageManagement.PowerShellCmdlets/Cmdlets/UpdatePackageCommand.cs

```diff
@@ -209,9 +209,14 @@ protected override void WarnIfParametersAreNotSupported()
                 // set nuget operation status to failed when an exception is thrown
                 _status = NuGetOperationStatus.Failed;
 
+                if (!string.IsNullOrEmpty(ex.Message))
```

Comment:
nit: should we have all this repeating logic (checking `ex.Message`, logging it and logging all issues on results) inside of a static utility method and just call it in all occurrences to avoid code duplication?

Reply:
The loggers are different, so the only common logic would be to check string null or empty.